processing/src/main/java/org/apache/druid/segment/LongColumnSerializerV2.java

```diff
@@ -121,12 +121,23 @@ public void open() throws IOException
   public void serialize(ColumnValueSelector<?> selector) throws IOException
   {
     if (selector.isNull()) {
-      nullRowsBitmap.add(rowCount);
-      writer.add(0L);
+      serializeNull();
     } else {
-      writer.add(selector.getLong());
+      serializeValue(selector.getLong());
     }
-    rowCount++;
+  }
+
+  public void serializeNull() throws IOException
```

Comment:
Please add java docs for these public methods.

Reply:
Added